qontract_api/qontract_api/integrations/openshift_namespaces/tasks.py

```
@@ -0,0 +1,121 @@
+@celery_app.task(bind=True, name="openshift_namespaces.reconcile", acks_late=True)
+@deduplicated_task(lock_key_fn=generate_lock_key, timeout=600)
+def reconcile_openshift_namespaces_task(
+    self: Any,
+    clusters: list[ClusterNamespaces],
+    *,
+    dry_run: bool = True,
+) -> OpenShiftNamespacesTaskResult:
+    """Reconcile openshift namespaces (background task)."""
+    request_id = self.request.id
+
+    try:
```

Ruff (0.15.20)

[warning] Try clause contains too many statements (12 > 5)

(PLW0717)